src/components/Popup/Popup.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import "./popup.css";
+import TextareaAutosize from 'react-textarea-autosize';
+
+function Popup({ trigger, setTrigger, card, updateCard }) {
```

Comment:
I prefer to make Popup component that will represent empty popup that will serve content from props.children.
And in thin case you should create something like EditCardForm component that will be served

Reply:
done